tests/DockerFileGeneratorTest.php

```diff
@@ -182,16 +161,7 @@ public static function generatorRequirementsProvider(): iterable
                 ],
             ],
             'box.phar',
-            <<<'Dockerfile'
-                FROM php:7.1-cli-alpine
-
-                RUN $(php -r '$extensionInstalled = array_map("strtolower", \get_loaded_extensions(false));$requiredExtensions = ["zlib"];$extensionsToInstall = array_diff($requiredExtensions, $extensionInstalled);if ([] !== $extensionsToInstall) {echo \sprintf("docker-php-ext-install %s", implode(" ", $extensionsToInstall));}echo "echo \"No extensions\"";')
-
-                COPY box.phar /box.phar
-
-                ENTRYPOINT ["/box.phar"]
-
-                Dockerfile,
+            '7.1-cli-alpine'
```

Comment:
same here, although from your changes the generated dockerfile would actually be simpler to write & read